src/Mod/Assembly/CommandCreateJoint.py

```diff
@@ -213,26 +221,184 @@ def GetResources(self):
                 "Assembly_CreateJointDistance",
                 "Create a Distance Joint: Fix the distance between the selected objects.",
             )
+            + "</p><p>"
+            + QT_TRANSLATE_NOOP(
+                "Assembly_CreateJointDistance",
+                "It can be used to create a lot of different joints based on the selection."
+                "For example a distance of 0 between a plane and a cylinder is a tangent. A distance of 0 between planes will make them coplanar. And so on.",
+            )
```

Comment:
This wording is a little casual. Consider something like:
```suggestion
                "Create one of several different joints based on the selection."
                "For example, a distance of 0 between a plane and a cylinder creates a tangent. A distance of 0 between planes will make them co-planar.",
            )
```